apps/pie-docs/src/designers/getting-started/best-practices.md

```diff
@@ -8,7 +8,7 @@ eleventyNavigation:
 
 Make sure you always use the styles defined in our Foundations libraries. This will help ensure you’re aligned with the PIE design system and enhance consistency across our products and files.
```

Comment:
super nit but should it be `Foundation libraries` instead of `Foundations libraries`?

Reply:
I think either could work as long as we're consistent

Reply:
agree - looks like it's referred to plurally in the text but is singular in the subtitle. Maybe changing that back to plural would be easiest to make it consistent